index.html

```diff
@@ -203,11 +213,13 @@ <h4>About me</h4>
 <!---------  contact form ------------------------------------->
 <form action="https://formspree.io/f/mqkoavny" method="post">
     <ul>
-        <li><h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2></li>
-        <li><input type="text" name="user-name" id="user-name" placeholder="Full name" required maxlength="30"></li>
-        <li><input type="email" name="email" id="email" placeholder="Email address" required></li>
-        <li><textarea name="message" id="message" maxlength="500" placeholder="Enter text here" required></textarea></li>
-        <li><button id="get-in-touch" name="get-in-touch">Get in touch</button></li>
+      <div class="contact-left"> <li><h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2></li>
+     </div>
+       <div class="contact-right"> <li><input type="text" name="user-name" id="user-name" placeholder="Full name" required maxlength="30"></li>
+            <li><input type="email" name="email" id="email" placeholder="Email address" required></li>
+            <li><textarea name="message" id="message" maxlength="500" placeholder="Enter text here" required></textarea></li>
+            <li><button id="get-in-touch" name="get-in-touch">Get in touch</button></li>
+        </div>
     </ul>
 </form>  
 
```

Comment:
- [ ] Kindly update the background of the footer to match the template. 🧐️

**Your Design**
![Screenshot from 2023-01-26 07-41-35](https://user-images.githubusercontent.com/17243325/214769136-030950e8-503c-47cb-a824-319c3f2b4b5d.png)

**Figma Design**
![Screenshot from 2023-01-26 07-41-11](https://user-images.githubusercontent.com/17243325/214769164-f44074f3-289f-44d2-b1a6-96e5e785b73c.png)